### DISCUSSION TITLE
I need direction

### DISCUSSION THREAD

**Discussion by dev-shivam-verma** [open-sauced/feedback#43] [Q&A]
Hi there, I am very new to these communities and open source, and a beginner android developer trying to upskill.
Can you please refer some very basic issues i can resolve and start contributing.

**Comment by jpmcb** [comment]
Hi @dev-shivam-verma - if you look for issues with labels "good first issue", those are tickets that the engineering team has marked as being appropriate for new comers and first timers to work on.

https://github.com/open-sauced/app/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

Make sure to read the `CONTRIBUTING.md` for the repo you're wanting to contribute to!!! And don't be afraid to ask for help!
